lib/assets/javascripts/turbograft/initializers.coffee

```diff
@@ -16,6 +16,8 @@ setupRemoteFromTarget = (target, httpRequestType, form = null) ->
   options =
     httpRequestType: httpRequestType
     httpUrl: httpUrl
+    forceScrollReset: TurboGraft.getTGAttribute(target, 'force-scroll-reset')
+    forceHeadAssetInsertion: TurboGraft.getTGAttribute(target, 'force-head-asset-insertion')
```

Comment:
It feels like these are always going to go together.  Is there situations in which `forceScrollReset != forceHeadAssetInertion`?